lib/channel-postmessage/src/index.ts

```diff
@@ -87,7 +87,11 @@ export class PostmsgTransport {
       }).filter(([k, v]) => typeof v !== 'undefined')
     );
 
-    const stringifyOptions = { ...(global.CHANNEL_OPTIONS || {}), ...c };
+    const stringifyOptions = {
+      ...{ allowFunction: true, maxDepth: 25 },
```

Comment:
Do we want to have an object that states what these are? like `const sensibleDefaults` or something? And maybe we can rename `c` to something that is more explanatory?